packages/relay/src/routes/voteRoute.ts

```diff
@@ -26,7 +26,7 @@ export default (
             if (voteAction == undefined) {
                 throw InvalidVoteActionError
             }
-            if (InvalidPublicSignalError == undefined) {
+            if (InvalidPublicSignalError == null) {
```

Comment:
`InvalidPublicSignalError` is always initialized first at `/InternalError.ts` file. Why do we check `InvalidPublicSignalError` to be null here?

Reply:
fixed